dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi3.0-messaging3.0.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.cdi3.0-messaging3.0
+visibility=private
+IBM-App-ForceRestart: install,uninstall
+IBM-Provision-Capability:\
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-3.0))",\
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.jakarta.messaging-3.0))"
+-bundles=com.ibm.ws.messaging.jms.2.0.cdi.jakarta
+IBM-Install-Policy: when-satisfied
+kind=noship
+edition=full
+WLP-Activation-Type: parallel
```

Comment:
The jakarta messaging features have not yet been enabled for parallel activation because the connector feature dependencies were also not enable.  As of yesterday the connector features are now enabled, so...  If you to enable this auto feature for parallel activation, then you should either enable all the jakarta messaging features in this PR or remove this WLP-Activation-Type header, and we'll attempt to enable the feature set in a separate branch.  We might see build breaks due to parallel activation, fyi.

Reply:
Removed.